terraform/variables.tf

```diff
@@ -28,6 +28,26 @@ variable "data_snapshot_s3_bucket" {
   default     = "univaf-data-snapshots"
 }
 
+variable "db_user" {
+  description = "The database user"
+  default     = "univaf"
+}
+
+variable "db_password" {
+  description = "The password for the database instance, filled via Terraform"
+  sensitive   = true
+}
+
+variable "db_instance" {
+  description = "The instance type for the DB. Reference: https://aws.amazon.com/rds/instance-types/"
+  default     = "db.t4g.small"
+}
+
+variable "db_size" {
+  description = "The storage size for the DB (in Gigabytes)"
+  default     = 30
```

Comment:
Was previously `240`, which was a result of dealing with an long-ago outage related to clearing old logs. We always meant to turn this down, but never got to it.

In Render, we have a 48 GB disk and have not used more than 12 GB of it in the past week, so this ought to be plenty.